LoRaEngine/modules/LoRaWanNetworkSrvModule/Logger/ILoggerExtensions.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan
+{
+    using Microsoft.Extensions.Logging;
+    using System;
+    using System.Collections.Generic;
+
+    public static class ILoggerExtensions
+    {
+        public const string DevEUIKey = "DevEUI";
+
+        public static IDisposable BeginDeviceScope(this ILogger logger, string devEUI) =>
```

Comment:
I think it could be also interesting to have a "StationScope" here. What do you think?

Reply:
Absolutely. As described in the comment, we can extend this. We have to be a bit careful not overloading every entry, on the console though. We will have the scopes include much more and that can be used in any way we want to. 

This is about having the base to replace the current logger .